lib/sports_data_api.rb

```diff
@@ -3,6 +3,7 @@
 require "rest_client"
 require "time"
 require "multi_json"
+require "json"
```

Comment:
Why `json` over `multi_json`? Is there something this is providing that `MultiJson` is not?